install_dependencies.sh

```diff
@@ -19,6 +19,7 @@ it installs the given dependencies:
   - curl library                            curl-7.62.0
   - openssl library                         openssl-1.1.1a
   - c-ares library                          c-ares-1.15.0
+  - boost library                           boost-1.72.0
 
 for linux:
 ./install_dependencies.sh linux
```

Comment:
Are elevated permissions needed for running this script (make outputs etc.,)? In that case, should we call out this script to be executed as a privileged user?

Reply:
Never mind, looks like the make install commands seem to target the local directory anyway...